challenge_3/cpp/blakebarnes00/src/main.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+void print_array(std::vector<int> array)
+{
+  for(int i = 0; i < array.size(); i++)
```

Comment:
Yes that's one way. I would prefer the reverse iterator or reverse index to not modify the parameter or use extra processing time. Here's some examples.
https://gist.github.com/erocs/eced45eb4af154da991ec64500bb46f8